pv211_utils/query_ordering.py

```diff
@@ -0,0 +1,53 @@
+ARQMATH_QUERIES = {
```

Comment:
Please, explain the purpose of the file (not just to me but as a docstring here).

Reply:
todo